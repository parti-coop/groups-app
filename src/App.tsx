import React from 'react';
import {StatusBar} from 'react-native';
import {createAppContainer, createSwitchNavigator} from 'react-navigation';
import AppNavigator from './AppNavigator';
import AuthLoading from './screens/AuthLoading';
import AuthMain from './screens/AuthMain';
import {StoreProvider} from './Store';
const AppContainer = createAppContainer(
  createSwitchNavigator(
    {
      AuthLoading,
      AppNavigator,
      AuthMain,
    },
    {
      initialRouteName: 'AuthLoading',
    },
  ),
);
export default () => {
  return (
    <StoreProvider>
      <StatusBar barStyle="dark-content" />
      <AppContainer />
    </StoreProvider>
  );
};
