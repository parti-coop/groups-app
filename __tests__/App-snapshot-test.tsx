import React from 'react';
import renderer from 'react-test-renderer';

import App from '../src/App';

it('renders correctly with defaults', () => {
  const button = renderer.create(<App />).toJSON();
  expect(button).toMatchSnapshot();
});
