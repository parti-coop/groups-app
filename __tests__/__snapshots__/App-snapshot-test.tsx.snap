// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View>
    <Text>
      어서오세요!
    </Text>
  </View>
</RCTSafeAreaView>
`;
